static/locales/en-US.yaml

```diff
@@ -928,3 +932,9 @@ Hashtag:
 Yes: Yes
 No: No
 Ok: Ok
+
+Grammar:
+  # adds a colon after a form label
+  ColonText: '{text}:'
+  # Index / Count
+  Index / Count: '{index} / {count}'
```

Comment:
I think these are similar to the separators

Reply:
I'll remove Index / Count and I'll add a colon to the resouces that need one (French uses `  :  ` (space in front and at end) and armenian uses `՝  ` instead of `: `)